### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by B0pol**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I have tested the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).
- [x] I agree to create a pull request for [NewPipe](https://github.com/TeamNewPipe/NewPipe) as soon as possible to make it compatible with the changed API.

### CODE REVIEW

File: extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/search/YoutubeMusicSearchExtractorTest.java [hunk 1]
```diff
@@ -153,8 +153,8 @@ public static void setUp() throws Exception {
 
     public static class CorrectedSearch extends DefaultSearchExtractorTest {
         private static SearchExtractor extractor;
-        private static final String QUERY = "duo lipa";
-        private static final String EXPECTED_SUGGESTION = "dua lipa";
+        private static final String QUERY = "nocopyrigh sounds";
```
Comment (TheAssassin) [inline extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/search/YoutubeMusicSearchExtractorTest.java:156]: Is this typo intentional? If yes, consider adding a comment.

**Review by TheAssassin — COMMENTED** [review]
Looks good. The tests should be restored with different channels, if possible. Consider opening an issue about this.

File: extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/search/YoutubeMusicSearchExtractorTest.java (hunk 1 above)
Comment (B0pol) [reply extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/search/YoutubeMusicSearchExtractorTest.java:156]: Yes, it is. look at the test name: "CorrectedSearch" and EXPECTED_SUGGESTION below. I don't think it needs a comment, the variable / class / function names says everything

**Comment by B0pol** [comment]
I opened https://github.com/TeamNewPipe/NewPipeExtractor/issues/443

**Review by TheAssassin — APPROVED** [review]
I think it's okay, but I'd wait for @TobiGr to approve it as well.

**Review by TobiGr — APPROVED** [review]
Thanks

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java [hunk 2]
```diff
@@ -768,11 +770,11 @@ private EmbeddedInfo getEmbeddedInfo() throws ParsingException, ReCaptchaExcepti
 
         } catch (IOException e) {
             throw new ParsingException(
-                    "Could load decryption code form restricted video for the Youtube service.", e);
+                    "Could load deobfuscation code form restricted video for the Youtube service.", e);
```
Comment (Stypox) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java:773]: Mmmh, this should probably be "Could not load deobfuscation code for restricted video"

Comment (B0pol) [reply extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java:773]: Sounds good. Feel free to open a pn

Comment (Stypox) [reply extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java:773]: #444

**Pull request merged**